packages/state-transition/src/cache/epochCache.ts

```diff
@@ -485,6 +493,12 @@ export class EpochCache {
       nextSyncCommitteeIndexed = new SyncCommitteeCacheEmpty();
     }
 
+    // Compute PTC for this epoch
+    let PTC: ValidatorIndex[][] = [];
```

Comment:
really not a fan of abbreviating this as PTC but I see it's done in the spec too

Reply:
Any suggestion?